frontend/src/component/feature/FeatureStrategy/FeatureStrategyEmpty/FeatureStrategyEmpty.tsx

```diff
@@ -23,12 +22,33 @@ interface IFeatureStrategyEmptyProps {
     environmentId: string;
 }
 
+const StyledContainer = styled('div')(({ theme }) => ({
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    justifyContent: 'center',
+    paddingTop: theme.spacing(2),
+}));
+
+const StyledTitle = styled('div')(({ theme }) => ({
+    fontSize: theme.fontSizes.bodySize,
```

Comment:
```suggestion
    fontSize: theme.typography.body1.fontSize,
```

Reply:
we don't have body1.fontSize in the entire codebase. Not sure what that is